### PR TITLE
Seal classes that should be sealed

### PR DESCRIPTION
**Pull request by willsb**
This PR was sitting on my machine, might as well submit it.

Very easy to review, but requires attention!

Squash whenever

### CODE REVIEW

File: Toggl.Daneel.Tests.UI/MainTests.cs [hunk 1]
```diff
@@ -15,10 +15,7 @@ public class MainTests
         [SetUp]
         public void BeforeEachTest()
         {
-            app = ConfigureApp
-                .iOS
-                .EnableLocalScreenshots()
-                .StartApp();
+            app = Configuration.GetApp();
```
Comment (amulware) [inline Toggl.Daneel.Tests.UI/MainTests.cs]: That's not part of what it says it does.

File: Toggl.Daneel.Tests.UI/OnboardingTests.cs [hunk 2]
```diff
@@ -5,15 +5,15 @@
...
+    public sealed class OnboardingTests
     {
         private iOSApp app;
 
         [SetUp]
         public void BeforeEachTest()
         {
             app = Configuration.GetApp();
-
+
```
Comment (amulware) [inline Toggl.Daneel.Tests.UI/OnboardingTests.cs]: Trailing whitespace.

File: Toggl.Daneel.Tests.UI/StartTimeEntryTests.cs [hunk 3]
```diff
@@ -15,10 +15,7 @@ public class StartTimeEntryTests
         [SetUp]
         public void BeforeEachTest()
         {
-            app = ConfigureApp
-                .iOS
-                .EnableLocalScreenshots()
-                .StartApp();
+            app = Configuration.GetApp();
```
Comment (amulware) [inline Toggl.Daneel.Tests.UI/StartTimeEntryTests.cs]: A thing.

File: Toggl.Daneel/Presentation/TogglPresenter.cs [hunk 4]
```diff
@@ -36,11 +36,11 @@ protected override void RegisterAttributeTypes()
         {
             base.RegisterAttributeTypes();
 
-            AttributeTypesToShowMethodDictionary.Add(typeof(NestedPresentationAttribute), ShowNestedViewController);
-            AttributeTypesToShowMethodDictionary.Add(typeof(ModalCardPresentationAttribute), ShowModalCardViewController);
+            AttributeTypesToShowMethodDictionary.Add(typeof(NestedPresentationAttribute), showNestedViewController);
+            AttributeTypesToShowMethodDictionary.Add(typeof(ModalCardPresentationAttribute), showModalCardViewController);
```
Comment (amulware) [inline Toggl.Daneel/Presentation/TogglPresenter.cs]: All these changes do something unrelated.

File: Toggl.Daneel/ViewSources/TimeEntriesLogViewSource.cs [hunk 5]
```diff
@@ -28,9 +28,9 @@ public TimeEntriesLogViewSource(UITableView tableView)
 
         public override UIView GetViewForHeader(UITableView tableView, nint section)
         {
-            var grouping = GetGroupAt(section);
+            var grouping = getGroupAt(section);
 
-            var cell = GetOrCreateHeaderViewFor(tableView);
+            var cell = getOrCreateHeaderViewFor(tableView);
```
Comment (amulware) [inline Toggl.Daneel/ViewSources/TimeEntriesLogViewSource.cs]: More unrelated changes.

File: Toggl.Foundation/Resources.Designer.cs [hunk 6]
```diff
@@ -16,7 +16,7 @@ namespace Toggl.Foundation {
     [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
     [System.Diagnostics.DebuggerNonUserCodeAttribute()]
     [System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
-    public class Resources {
+    public sealed class Resources {
```
Comment (amulware) [inline Toggl.Foundation/Resources.Designer.cs]: In the Designer.cs?

File: Toggl.Daneel/Presentation/TogglPresenter.cs (hunk 4 above)
Comment (willsb) [reply Toggl.Daneel/Presentation/TogglPresenter.cs]: They are not. You can't seal a class with virtual protected members.

File: Toggl.Foundation/Resources.Designer.cs (hunk 6 above)
Comment (willsb) [reply Toggl.Foundation/Resources.Designer.cs]: That's mildly wrong, yes

**Comment by willsb** [comment]
@simonrozsival These last three classes aren't even included in the project and should be 🔥
